src/corelibs/U2View/src/ov_msa/MaEditor.cpp

```diff
@@ -463,7 +463,7 @@ void MaEditor::updateFontMetrics() {
     const int minimumSafeFontPointSize = 8;  // This value was historically used in UGENE as minimum with no known issues.
     QFont fontToEstimate = font;
     int estimatedMinimumFontPointSize = minimumSafeFontPointSize;  // Start with a safe value and estimate smaller values.
-    while (fontToEstimate.pointSize() > 0) {
+    while (fontToEstimate.pointSize() > 1) {
```

Comment:
The error sign is `QFont::setPointSize: Point size <= 0 (0), must be greater than 0`.
The point size here is already greater than 0, you made them greater than 1. I don't see how it helps.

Reply:
Could you please check the algorithm beyond the changed line? The only reason to check the value for 1 here is to decrease it by -1 3 lines later.